plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveHadoop.java

```diff
@@ -72,6 +77,17 @@ protected void setupContainer()
                         runCmd));
     }
 
+    @Override
+    protected void startContainer()
+    {
+        super.startContainer();
+        log.info(format(
+                "Hive container started with addresses for SOCKS proxy: %s; Metastore: %s; HiveServer2: %s",
+                getMappedHdfsSocksProxy(),
+                getHiveMetastoreEndpoint(),
+                getHiveServerEndpoint()));
```

Comment:
How about using logger format instead of `java.lang.String.format`?

Reply:
good point :)